pkg/mkverification-raw-efi/build.yml

```diff
@@ -0,0 +1,3 @@
+org: lfedge
+image: mkverification-raw-efi
+network: yes
```

Comment:
You should be able to use ADD to get all your dependencies and let network disabled here....

Reply:
I need a lot of packages that we do not include in our cache of our base alpine image. We can either add them to our cache and create a new version of lfedge/eve-alpine, or do what I did. I prefer what I did because it is self-contained and does not affect other images we build. However, I do not have a strong opinion about it, so If we think changing lfedge/eve-alpine is a better approach I will do it.

Reply:
see above for motication to use ADD commands.